_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Hello^0^I am Yini He!
+categories: blog
+excerpt: An introduction to the Whitaker lab's newest member
+image:
+  feature:
+link:
+date: 2019-06-20
+modified:
+share: true
+author: Yini-He
+---
+
+## Who am I
+
+--I am who I am（我就是我）
+
+My name is Yini He（何旖旎）.I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student,jointly funded by Chinese Scholarship Council (CSC) and the Newton Fund.I am enthusiastic about  the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.
+
+
+## Where I come here  
```

Comment:
Where I come here doesn't make grammatical sense. Do you mean "Where I come from"?